shelley-ma/shelley-ma-test/cddl-files/shelley-ma.cddl

```diff
@@ -211,10 +211,10 @@ transaction_metadatum_label = uint
 
 transaction_metadata =
   { * transaction_metadatum_label => transaction_metadatum }
-  / [{ ? 0: { * transaction_metadatum_label => transaction_metadatum }
-     , ? 1: [ * native_script ]
+  / [ 0: { * transaction_metadatum_label => transaction_metadatum }
+     , 1: [ * native_script ]
      ; other types of metadata...
-    }]
+    ]
```

Comment:
If you're changing it to a dense encoding then the labels `0:` and `1:` are just informative rather than part of the encoding. IIRC some code generators emit these as names. So these could be more descriptive.